packages/core/package.json

```diff
@@ -69,69 +46,125 @@
       "email": "trenaryja@gmail.com"
     }
   ],
-  "license": "MIT",
+  "sideEffects": false,
+  "type": "module",
+  "exports": {
+    ".": {
+      "import": {
+        "default": "./dist/pcln-design-system.js",
+        "types": "./dist/index.d.ts"
+      },
+      "require": {
+        "default": "./dist/pcln-design-system.umd.cjs",
+        "types": "./dist/index.d.ts"
+      }
+    }
+  },
+  "main": "./dist/pcln-design-system.umd.cjs",
+  "module": "./dist/pcln-design-system.js",
+  "types": "dist/index.d.ts",
+  "files": [
+    "dist",
+    "types"
+  ],
+  "scripts": {
+    "vite:dev": "vite",
+    "build": "vite build",
+    "build:visualize": "vite-bundle-visualizer",
+    "build:visualize:sunburst": "vite-bundle-visualizer --template sunburst",
+    "build:visualize:network": "vite-bundle-visualizer --template network",
+    "build:visualize:data": "vite-bundle-visualizer --template raw-data --output './vite-bundle-visualizer/stats.json'",
+    "build:visualize:list": "vite-bundle-visualizer --template list --output './vite-bundle-visualizer/stats.yaml'",
+    "build:watch": "vite build --watch",
+    "lint": "eslint src --ext ts,tsx --report-unused-disable-directives --max-warnings 0",
+    "vite:preview": "vite preview",
```

Comment:
@sdalonzo I tried running `vite:dev` and `vite:preview` but they don't seem to do anything. Should we remove those to avoid confusion?

Reply:
done